### PR TITLE
migrate set function

### PR DESCRIPTION
**Pull request by maximtop**
This PR fixes: https://github.com/date-fns/date-fns-typescriptify/issues/120

### CODE REVIEW

**Review by rikkalo — APPROVED** [review]
Thank you for your contribution 👍

File: src/set/index.ts [hunk 1]
```diff
@@ -3,6 +3,16 @@ import setMonth from '../setMonth/index'
...
+interface Values {
+  year?: number;
+  month?: number;
+  date?: number;
+  hours?: number;
+  minutes?: number;
+  seconds?: number;
+  milliseconds?: number;
+}
+
```
Comment (kossnocorp) [inline src/set/index.ts:15]: Could you please move it to `src/types.ts`

**Review by kossnocorp — CHANGES_REQUESTED** [review]
Thank you a lot, please move the interface to shared types cc @rikkalo

File: src/set/index.ts (hunk 1 above)
Comment (maximtop) [reply src/set/index.ts:6]: I've moved

**Review by kossnocorp — APPROVED** [review]
🙏

**Pull request merged**